scripts/query_mlwh.py

```diff
@@ -0,0 +1,152 @@
+import os
+import MySQLdb
+import pandas as pd
+
+# depend on IRODS structure 
+
+def fetch_warehouse_sample_meta(db_config_file, sample_list, id_type='name'):
+    '''
+    Query mlwh database using list of sample IDs
+    
+    Parameters:
+    db_config_file - path to mlwh config
+    sample_list - list of sample IDs, 
+    id_type - type of sample ID, valid values: 
+        - 'name' (corresponds to 'sanger_sample_id')
+        - 'supplier_name' (sample name in SequenceScape manifest, sometimes corresponds to 'derived_sample_id')
+        - 'accession_number' (corresponds to 'ebi_sample_acc')
+    
+    Return pandas dataframe of sample metadata
+    '''
```

Comment:
No objection to merging this as-is if you are pressed for time, but FWIW I like to follow the [numpy docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard) for docstrings.